### ISSUE TITLE
Equip feature has gone wonky

### ISSUE THREAD

**Issue by maxell45146** [TheCyberBrick/The-Betweenlands#677]
![2018-08-01_21 05 12](https://user-images.githubusercontent.com/2869265/43556976-5d4d1e28-95d0-11e8-85d5-1fd6e391d570.png)

Was trying out the equip option for the first time since installing the mod. Lurker pouch was a nice thing to have. Went to open it though after equipping a green gem amulet and things went sideways. Basically now when attempting to use the pouch instead of opening its normal window it literally takes over my inventory. All my gear disappears and any items I had in the pouch now are in my main inventory. To make it worse I thought to unequip everything to try fixing it. No good, when pressing C now it shows a small circle and nothing else. I've attached a screenshot. 

Currently playing this on multiplayer, I went ahead and made sure I was on the current version, that the old core mod was removed and that the server version was being used on the server. Same issue. 

One thing I did notice is that if I log off the server then back on my items will return but if I try using the pouch or equipment the same issue appears.   

Mods:
  ActuallyAdditions-1.12.2-r135
  AkashicTome-1.2-10
  angelRingToBauble-1.12-0.3.1.50+d4e654e
  AppleSkin-mc1.12-1.0.9
  appliedenergistics2-rv6-alpha-2
  Aroma1997Core-1.12.2-2.0.0.0.b150
  Aroma1997s-Dimensional-World-1.12-1.3.0.0.b37
  astralsorcery-1.12.2-1.9.3
  AutoRegLib-1.3-17
  badwithernocookiereloaded-1.12.2-2.12
  Baubles-1.12-1.5.2
  bdlib-1.14.3.12-mc1.12.2
  bedpatch-2.2-1.12.2
  BetterAdvancements-1.12.2-0.0.5.17
  BetterBuildersWands-1.12-0.11.1.245+69d0d70
  BetterWithEngineering-1.6
  BetterWithMods-1.12-2.3.3
  BiblioCraft[v2.4.5][MC1.12.2]
  binnie-mods-1.12.2-2.5.0.165
  BiomesOPlenty-1.12.2-7.0.1.2398-universal
  blockcraftery-0.1.3
  BloodArsenal-1.12.2-2.1.0-24
  BloodMagic-1.12.2-2.3.1-99
  bonsaitrees-1.0.5-b77
  Bookshelf-1.12.2-2.3.544
  Botania r1.10-354
  BrandonsCore-1.12.2-2.4.3.168-universal
  BuildingGadgets-1.5.1
  careerbees-0.4.0
  Chameleon-1.12-4.1.3
  ChickenChunks-1.12.2-2.4.1.71-universal
  Chisel-MC1.12.2-0.2.0.31
  chiselsandbits-14.19
  ClocheCall-1.1.1
  CodeChickenLib-1.12.2-3.2.0.345-universal
  CoFHCore-1.12.2-4.5.2.19-universal
  CoFHWorld-1.12.2-1.2.0.5-universal
  ColossalChests-1.12.2-1.6.11
  comforts-1.12.2-1.1.2
  CommonCapabilities-1.12-1.4.0
  compactmachines3-1.12.2-3.0.12-b215
  CompactSolars-1.12.2-5.0.17.340-universal
  Controlling-3.0.6
  CookingForBlockheads_1.12.2-6.3.26
  CraftTweaker2-1.12-4.1.9
  CTM-MC1.12-0.2.3.12
  CustomMainMenu-MC1.12.2-2.0.8
  CyclopsCore-1.12.2-0.11.8
  DankNull-1.12-1.4.33
  DarkUtils-1.12.2-1.8.211
  deepresonance-1.12-1.7.0
  DefaultOptions_1.12.2-9.2.7
  diethopper-1.1
  Draconic-Evolution-1.12.2-2.3.12.297-universal
  DummyCoreUnofficial-2.4.112.3
  ElecCore-1.12.2-1.8.434
  ElevatorMod-1.12.2-1.3.6
  elulib-0.1.12
  embers-0.230
  EnderCore-1.12.2-0.5.35
  EnderIO-1.12.2-5.0.29
  EnderStorage-1.12.2-2.4.2.126-universal
  environmentaltech-1.12.2-2.0.10a
  EvilCraft-1.12.2-0.10.61
  Exchangers-1.12.2-2.7.2
  ExpandedEquivalence-1.12.2-10a
  ExtraBotany-r1.1-30
  extrautils2-1.12-1.8.1
  ExtremeReactors-1.12.2-0.4.5.48
  FarmingForBlockheads_1.12.2-3.1.21
  FastWorkbench-1.12.2-1.3.2
  flatcoloredblocks-mc1.12-6.6
  foamfix-0.9.9.1-1.12.2-anarchy
  forestry_1.12.2-5.8.0.290
  Forgelin-1.6.0
  ForgeMultipart-1.12.2-2.4.2.58-universal
  FTBLib-5.0.2.14
  FTBUtilities-5.0.2.14
  funky-locomotion-1.12.2-1.1.1
  gendustry-1.6.5.8-mc1.12.2
  generators-0.9.20.12-mc1.12.2
  gravestone-1.10.1
  Guide-API-1.12-2.1.7-62
  HammerCore-1.12.2-2.0.0.1
  iceandfire-1.5.2
  iChunUtil-1.12.2-7.1.4
  ImmersiveCables-1.12.2-1.3.1
  ImmersiveEngineering-0.12-84
  immersivepetroleum-1.12.2-1.1.9
  immersivetech-1.12-1.3.10
  ImprovedBackpacks-1.12.2-1.3.0.0
  industrialcraft-2-2.8.73-ex112
  industrialforegoing-1.12.2-1.10.7-204
  IntegratedDynamics-1.12.2-0.11.11
  IntegratedTunnels-1.12.2-1.5.5
  IntegrationForegoing-1.12.2-1.7.3
  inventorysorter-1.12.2-1.13.3+57
  IronBackpacks-1.12.2-3.0.7-10
  ironchest-1.12.2-7.0.40.824
  JEHC-1.12.2-1.6.1.0
  jei_1.12.2-4.11.0.206
  jeibees-0.9.0.5-mc1.12.2
  jeiintegration_1.12.2-1.5.1.36
  journeymap-1.12.2-5.5.2
  JurassiCraft-1.12.2-2.1.14
  JustEnoughResources-1.12.2-0.8.6.28
  llibrary-1.7.14-1.12.2
  LLOverlayReloaded-1.1.4-mc1.12
  lostcities-1.12-2.0.7
  LunatriusCore-1.12.2-1.2.0.42-universal
  MagicalPsi-1.3
  MagicBees-1.12.2-3.1.9
  Mantle-1.12-1.3.2.24
  mcjtylib-1.12-3.0.3
  MCMultiPart-2.5.1
  meecreeps-1.12-1.2.1
  Mercurius-1.12.2
  minetogether-1.10.2-1.8.7
  MobGrindingUtils-0.3.6
  modnametooltip_1.12.2-1.10.0
  modtweaker-4.0.12
  Morph-1.12.2-7.1.2
  Morpheus-1.12-3.3.2
  Morph-o-Tool-1.2-16
  MouseTweaks-2.8-mc1.12.1
  MrTJPCore-1.12.2-2.1.2.34-universal
  MTLib-3.0.4
  mystcraft-1.12.2-0.13.6.00
  NetherPortalFix_1.12.1-5.3.13
  Networks+Manager-12.0.3
  NoMoreRecipeConflict-0.11(1.12.2)
  NotEnoughIDs-1.5.4.2
  notenoughwands-1.12-1.7.0
  OpenComputers-MC1.12.2-1.7.2.67
  p455w0rdslib-1.12-2.0.29
  Pam's HarvestCraft 1.12.2u
  Placebo-1.12.2-1.3.4
  PortalGun-1.12.2-7.0.2
  practicallogistics2-1.12.2-3.0.4
  prefab-1.3.0.24
  ProjectE-1.12-PE1.3.1B
  ProjectRed-1.12.2-4.9.0.86-Base
  ProjectRed-1.12.2-4.9.0.86-compat
  ProjectRed-1.12.2-4.9.0.86-fabrication
  ProjectRed-1.12.2-4.9.0.86-integration
  ProjectRed-1.12.2-4.9.0.86-lighting
  ProjectRed-1.12.2-4.9.0.86-mechanical
  ProjectRed-1.12.2-4.9.0.86-world
  Psi-r1.1-59
  QuickLeafDecay-MC1.12.1-1.2.4
  randompatches-1.12.2-1.4.1.1
  RandomThings-MC1.12.2-4.2.0
  rangedpumps-0.5
  realfilingcabinet-1.12.1-0.1.57
  reauth-3.6.0
  RebornCore-1.12.2-3.9.2.319-universal
  RebornStorage-1.12.2-3.2.1.63
  RedstoneArsenal-1.12.2-2.5.1.13-universal
  RedstoneFlux-1.12-2.0.2.3-universal
  refinedstorage-1.6.2
  refinedstorageaddons-0.4.1
  ResourceLoader-MC1.12.1-1.5.3
  rftools-1.12-7.55
  rftoolsctrl-1.12-1.9.0
  rftoolsdim-1.12-5.60
  rftoolspower-1.12-1.1.1
  rftpwr_oc-0.2
  roots-2-0.104
  Schematica-1.12.2-1.8.0.169-universal
  Schematics-1.12.2.8
  SimplyJetpacks2-1.12.2-2.2.4.34
  sonarcore-1.12.2-5.0.12
  Soot-0.15
  stg-1.12.2-1.2.3
  StorageDrawers-1.12.2-5.3.6
  StorageDrawersExtras-1.12-3.1.0
  TConstruct-1.12.2-2.10.1.87
  Tesla-1.12.2-1.0.63
  tesla-core-lib-1.12.2-1.0.15
  Thaumcraft-1.12.2-6.1.BETA19
  ThaumicAdditions-1.12.2-3r
  thaumicbases-3.1.255.2r-ForTC6B17
  ThaumicEquivalence-1.12.2-1.0.5
  ThaumicInventoryScanning_1.12.2-2.0.10
  ThaumicJEI-1.12.2-1.4.0-14
  thaumicperiphery-0.2.4
  thaumictinkerer-1.12.2-5.0-51a5074
  TheBetweenlands-3.3.11-universal
  theoneprobe-1.12-1.4.23
  ThermalCultivation-1.12.2-0.3.0.7-universal
  ThermalDynamics-1.12.2-2.5.1.14-universal
  ThermalExpansion-1.12.2-5.5.0.29-universal
  ThermalFoundation-1.12.2-2.5.0.19-universal
  ThermalInnovation-1.12.2-0.3.0.7-universal
  TipTheScales-1.12.2-1.0.1
  Toast Control-1.12.2-1.6.0
  ToolBelt-1.12.0-1.7.1
  topaddons-1.12.2-1.6.0
  Translocators-1.12.2-2.5.0.70-universal
  twilightforest-1.12.2-3.7.424-universal
  UniDict-1.12.2-2.5b
  valkyrielib-1.12.2-2.0.10b
  WanionLib-1.12.2-1.5
  Waystones_1.12.2-4.0.54
  WirelessCraftingTerminal-1.12.2-3.9.64
  worldedit-forge-mc1.12-6.1.8-dist
  xnet-1.12-1.7.3
  zerocore-1.12-0.1.2.1

**Comment by maxell45146** [comment]
Update
Kept messing and it seems that if I create another pouch I'm able to equip it and the first pouch, eventually will end up back in my inventory. Still not sure what is causing the issue though but at least I have a work around.

**Comment by TheCyberBrick** [comment]
That's a very bizarre problem. I've seen other people report something similar with other backpacks too so I suspect it's another mod causing this problem. So far I've never run into this issue, but I'll check the code again and see if I can find something that could cause this

**Comment by maxell45146** [comment]
Thought I would add a update. It appears that no aspects are being recorded in the Herblore  Book aswell. Tried multiple items, each one says their is nothing left to discover but each page is still blank. Also, holding shift if post to show the aspect associated with the item after discovering it but that isnt appearing as well. At this point I'm wondering if it has something to do with the dimension I'm in when attempting all this? I've updated mods to the latest and forge as well. 

The issue also exists for any rings I try to equip. After equipping a ring some time later I will notice that the symbol next to the toolbar is gone and so is the effects. Its almost like somehow it loses the item (ring or pouch) and can only be retrieved by equipping and unequipping another item to take its place. So the items arent being lost or sent to $null but only retrieveable after using the system to equip another item to force the current one out of its location. ugh... 

List of Mods after updating.
ActuallyAdditions-1.12.2-r135
  AkashicTome-1.2-10
  angelRingToBauble-1.12-0.3.1.50+d4e654e
  AppleSkin-mc1.12-1.0.9
  appliedenergistics2-rv6-alpha-2
  Aroma1997Core-1.12.2-2.0.0.0.b150
  Aroma1997s-Dimensional-World-1.12-1.3.0.0.b37
  astralsorcery-1.12.2-1.9.3
  AutoRegLib-1.3-17
  badwithernocookiereloaded-1.12.2-2.12
  Baubles-1.12-1.5.2
  bdlib-1.14.3.12-mc1.12.2
  bedpatch-2.2-1.12.2
  BetterAdvancements-1.12.2-0.0.5.17
  BetterBuildersWands-1.12-0.11.1.245+69d0d70
  BetterWithEngineering-1.6
  BetterWithMods-1.12-2.3.3
  BiblioCraft[v2.4.5][MC1.12.2]
  binnie-mods-1.12.2-2.5.0.165
  BiomesOPlenty-1.12.2-7.0.1.2398-universal
  blockcraftery-0.1.3
  BloodArsenal-1.12.2-2.1.0-24
  BloodMagic-1.12.2-2.3.1-99
  bonsaitrees-1.0.5-b77
  Bookshelf-1.12.2-2.3.544
  Botania r1.10-354
  BrandonsCore-1.12.2-2.4.3.168-universal
  BuildingGadgets-1.5.1
  careerbees-0.4.0
  Chameleon-1.12-4.1.3
  ChickenChunks-1.12.2-2.4.1.73-universal
  Chisel-MC1.12.2-0.2.0.31
  chiselsandbits-14.19
  ClocheCall-1.1.1
  CodeChickenCore-1.10.2-2.4.1.102-universal
  CodeChickenLib-1.12.2-3.2.1.349-universal
  CoFHCore-1.12.2-4.5.3.20-universal
  CoFHWorld-1.12.2-1.2.0.5-universal
  ColossalChests-1.12.2-1.6.11
  comforts-1.12.2-1.1.2
  CommonCapabilities-1.12-1.4.0
  compactmachines3-1.12.2-3.0.12-b215
  CompactSolars-1.12.2-5.0.17.340-universal
  Controlling-3.0.6
  CookingForBlockheads_1.12.2-6.3.26
  CraftTweaker2-1.12-4.1.9
  CTM-MC1.12-0.2.3.12
  CustomMainMenu-MC1.12.2-2.0.8
  CyclopsCore-1.12.2-0.11.8
  DankNull-1.12-1.4.33
  DarkUtils-1.12.2-1.8.211
  deepresonance-1.12-1.7.0
  DefaultOptions_1.12.2-9.2.7
  diethopper-1.1
  Draconic-Evolution-1.12.2-2.3.12.297-universal
  DummyCoreUnofficial-2.4.112.3
  ElecCore-1.12.2-1.8.434
  ElevatorMod-1.12.2-1.3.6
  elulib-0.1.12
  embers-0.230
  EnderCore-1.12.2-0.5.37
  EnderIO-1.12.2-5.0.30
  EnderStorage-1.12.2-2.4.5.135-universal
  environmentaltech-1.12.2-2.0.10a
  EvilCraft-1.12.2-0.10.61
  Exchangers-1.12.2-2.7.2
  ExpandedEquivalence-1.12.2-10a
  ExtraBotany-r1.1-30
  extrautils2-1.12-1.8.1
  ExtremeReactors-1.12.2-0.4.5.48
  FarmingForBlockheads_1.12.2-3.1.21
  FastWorkbench-1.12.2-1.3.2
  flatcoloredblocks-mc1.12-6.6
  foamfix-0.9.9.1-1.12.2-anarchy
  forestry_1.12.2-5.8.0.290
  Forgelin-1.6.0
  ForgeMultipart-1.12.2-2.5.0.69-universal
  FTBLib-5.0.2.14
  FTBUtilities-5.0.2.14
  funky-locomotion-1.12.2-1.1.1
  gendustry-1.6.5.8-mc1.12.2
  generators-0.9.20.12-mc1.12.2
  gravestone-1.10.1
  Guide-API-1.12-2.1.7-62
  HammerCore-1.12.2-2.0.1.2
  iceandfire-1.5.2
  iChunUtil-1.12.2-7.1.4
  ImmersiveCables-1.12.2-1.3.1
  ImmersiveEngineering-0.12-84
  immersivepetroleum-1.12.2-1.1.9
  immersivetech-1.12-1.3.10
  ImprovedBackpacks-1.12.2-1.3.0.0
  industrialcraft-2-2.8.73-ex112
  industrialforegoing-1.12.2-1.10.7-204
  IntegratedDynamics-1.12.2-0.11.11
  IntegratedTunnels-1.12.2-1.5.5
  IntegrationForegoing-1.12.2-1.7.3
  inventorysorter-1.12.2-1.13.3+57
  IronBackpacks-1.12.2-3.0.7-10
  ironchest-1.12.2-7.0.40.824
  JEHC-1.12.2-1.6.1.0
  jei_1.12.2-4.11.0.206
  jeibees-0.9.0.5-mc1.12.2
  jeiintegration_1.12.2-1.5.1.36
  journeymap-1.12.2-5.5.2
  JurassiCraft-1.12.2-2.1.14
  JustEnoughResources-1.12.2-0.8.6.28
  llibrary-1.7.14-1.12.2
  LLOverlayReloaded-1.1.4-mc1.12
  lostcities-1.12-2.0.7
  LunatriusCore-1.12.2-1.2.0.42-universal
  MagicalPsi-1.3
  MagicBees-1.12.2-3.1.9
  Mantle-1.12-1.3.2.24
  mcjtylib-1.12-3.0.3
  MCMultiPart-2.5.1
  meecreeps-1.12-1.2.1
  Mercurius-1.12.2
  minetogether-1.10.2-1.8.7
  MobGrindingUtils-0.3.6
  modnametooltip_1.12.2-1.10.0
  modtweaker-4.0.12
  Morph-1.12.2-7.1.2
  Morpheus-1.12-3.3.2
  Morph-o-Tool-1.2-16
  MouseTweaks-2.8-mc1.12.1
  MrTJPCore-1.12.2-2.1.2.34-universal
  MTLib-3.0.4
  mystcraft-1.12.2-0.13.6.00
  NetherPortalFix_1.12.1-5.3.13
  Networks+Manager-12.0.3
  NoMoreRecipeConflict-0.11(1.12.2)
  NotEnoughIDs-1.5.4.2
  notenoughwands-1.12-1.7.0
  OpenComputers-MC1.12.2-1.7.2.67
  p455w0rdslib-1.12-2.0.29
  Pam's HarvestCraft 1.12.2u
  Placebo-1.12.2-1.3.4
  PortalGun-1.12.2-7.0.2
  practicallogistics2-1.12.2-3.0.4
  prefab-1.3.0.24
  ProjectE-1.12-PE1.3.1B
  ProjectRed-1.12.2-4.9.0.86-Base
  ProjectRed-1.12.2-4.9.0.86-compat
  ProjectRed-1.12.2-4.9.0.86-fabrication
  ProjectRed-1.12.2-4.9.0.86-integration
  ProjectRed-1.12.2-4.9.0.86-lighting
  ProjectRed-1.12.2-4.9.0.86-mechanical
  ProjectRed-1.12.2-4.9.0.86-world
  Psi-r1.1-59
  QuickLeafDecay-MC1.12.1-1.2.4
  randompatches-1.12.2-1.4.1.1
  RandomThings-MC1.12.2-4.2.0
  rangedpumps-0.5
  realfilingcabinet-1.12.1-0.1.57
  reauth-3.6.0
  RebornCore-1.12.2-3.9.3.324-universal
  RebornStorage-1.12.2-3.2.1.63
  RedstoneArsenal-1.12.2-2.5.1.13-universal
  RedstoneFlux-1.12-2.0.2.3-universal
  refinedstorage-1.6.3
  refinedstorageaddons-0.4.1
  ResourceLoader-MC1.12.1-1.5.3
  rftools-1.12-7.55
  rftoolsctrl-1.12-1.9.0
  rftoolsdim-1.12-5.60
  rftoolspower-1.12-1.1.1
  rftpwr_oc-0.2
  roots-2-0.104
  Schematica-1.12.2-1.8.0.169-universal
  Schematics-1.12.2.8
  SimplyJetpacks2-1.12.2-2.2.7.45
  sonarcore-1.12.2-5.0.12
  Soot-0.15
  stackable-1.12.2-1.0.0
  stg-1.12.2-1.2.3
  StorageDrawers-1.12.2-5.3.6
  StorageDrawersExtras-1.12-3.1.0
  TConstruct-1.12.2-2.10.1.87
  Tesla-1.12.2-1.0.63
  tesla-core-lib-1.12.2-1.0.15
  Thaumcraft-1.12.2-6.1.BETA19
  ThaumicAdditions-1.12.2-3r
  thaumicbases-3.1.255.2r-ForTC6B17
  ThaumicEquivalence-1.12.2-1.0.5
  ThaumicInventoryScanning_1.12.2-2.0.10
  ThaumicJEI-1.12.2-1.4.0-14
  thaumicperiphery-0.2.4
  thaumictinkerer-1.12.2-5.0-51a5074
  TheBetweenlands-3.3.11-universal
  theoneprobe-1.12-1.4.23
  ThermalCultivation-1.12.2-0.3.0.7-universal
  ThermalDynamics-1.12.2-2.5.1.14-universal
  ThermalExpansion-1.12.2-5.5.0.29-universal
  ThermalFoundation-1.12.2-2.5.0.19-universal
  ThermalInnovation-1.12.2-0.3.0.7-universal
  TipTheScales-1.12.2-1.0.1
  Toast Control-1.12.2-1.6.0
  ToolBelt-1.12.0-1.7.1
  topaddons-1.12.2-1.6.0
  Translocators-1.12.2-2.5.1.77-universal
  twilightforest-1.12.2-3.7.424-universal
  UniDict-1.12.2-2.5b
  valkyrielib-1.12.2-2.0.10b
  WanionLib-1.12.2-1.5
  Waystones_1.12.2-4.0.54
  WirelessCraftingTerminal-1.12.2-3.9.64
  worldedit-forge-mc1.12-6.1.8-dist
  WR-CBE-1.12.2-2.3.1.29-universal
  xnet-1.12-1.7.3
  zerocore-1.12-0.1.2.1
